From 609a918974843de1b3f7655340d1a4c74627fb04 Mon Sep 17 00:00:00 2001
From: Heba Aamer <heba93aamer@gmail.com>
Date: Tue, 27 Jan 2015 22:33:03 +0200
Subject: [PATCH 162/193] staging: fbtft: fix space required after that ';'

This patch fixes the following checkpatch.pl error:
fix space required after that ';'

Signed-off-by: Heba Aamer <heba93aamer@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/staging/fbtft/fb_pcd8544.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/staging/fbtft/fb_pcd8544.c b/drivers/staging/fbtft/fb_pcd8544.c
index c4f2a5b..a5f0ad4 100644
--- a/drivers/staging/fbtft/fb_pcd8544.c
+++ b/drivers/staging/fbtft/fb_pcd8544.c
@@ -117,10 +117,10 @@ static int write_vmem(struct fbtft_par *par, size_t offset, size_t len)
 
 	fbtft_par_dbg(DEBUG_WRITE_VMEM, par, "%s()\n", __func__);
 
-	for (x=0;x<84;x++) {
-		for (y=0;y<6;y++) {
+	for (x=0; x<84; x++) {
+		for (y=0; y<6; y++) {
 			*buf = 0x00;
-			for (i=0;i<8;i++) {
+			for (i=0; i<8; i++) {
 				*buf |= (vmem16[(y*8+i)*84+x] ? 1 : 0) << i;
 			}
 			buf++;
-- 
2.1.4

